casper/src/main/scala/coop/rchain/casper/PrettyPrinter.scala

```diff
@@ -30,6 +30,8 @@ object PrettyPrinter {
       s"Block #${postState.blockNumber} (${buildString(b.blockHash)}) " +
         s"-- Sender ID ${buildString(b.sender)} " +
         s"-- M Parent Hash ${buildString(mainParent)} " +
+        s"-- M Parents ${header.parentsHashList.map(PrettyPrinter.buildString).mkString(";")} " +
```

Comment:
I know I created those but do you think that it should stay? this will add heft to the logs

Reply:
Yeah we can remove this